src/routes/+layout.svelte

```diff
@@ -123,7 +160,18 @@
     {siteTitle}
     bind:navMenuExpanded
   />
-  <slot />
+  {#if previewAuthenticationError}
+    <div class="usa-section usa-prose grid-container">
+      <h1>{previewAuthenticationError.status}</h1>
+      <p>{previewAuthenticationError.message}</p>
+      <p>
+        (Do you need to <LoginLink {...loginLinkProps}>log in</LoginLink> or visit the
+        <a href={$page.url.toString()}>public version of the page</a>?)
```

Comment:
`$page.url` still includes `?preview` at the end of it. That needs to get stripped out to make sure it's a functional link.